azerothcore/azerothcore-wotlk
# fix(DB/creature): Rework AQ40 Qiraji Champion packs after Twin Emperors.

## Changes Proposed:
- Removes old guids
- Adds the 8 spawns of Qiraji Champion and their waypoints
- Adds the other members of the packs, with spawns randomized between Qiraji Brainwasher and Qiraji Slayer

## Issues Addressed:

- Closes unopened issue.

## SOURCE:


## Tests Performed:

- Tested ingame

## How to Test the Changes:


1. `.go c 88000`
2. observe the 8 packs of Qiraji Champions
3. see if they pull together, see if they correctly follow paths

## Known Issues and TODO List:


- [ ] I couldn't get points_1 and points_2 to work, no matter what values I used.
- [ ] Without the proper tools I haven't managed to figure out the correct spawn points for the formation members, so I  just spawned them on top of the leader :c 


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
